packages/clerk-js/src/core/clerk.ts

```diff
@@ -1350,4 +1352,27 @@ export default class Clerk implements ClerkInterface {
       throw new Error('ClerkJS components are not ready yet.');
     }
   }
+
+  #shouldRedirectFAPIInitiatedFlow = async (): Promise<boolean> => {
```

Comment:
Should we rename this since it might trigger a navigation?

Reply:
Renamed back to `redirectFAPIInitiatedFlow` which I think makes more sense, lmk what you think!